CenterForOpenScience/ember-osf-web
# [ENG-835] use subjects widget in registries overview

- Ticket: [ENG-835]
- Feature flag: n/a

## Purpose

Use Subjects::Widget and Subjects::Display for subject editable field in Registries overview page.

## Summary of Changes

## Changed
- Components
    - `registries/registries-metadata`
        - use `Subjects::Widget` and `Subjects::Display` (and related managers) for subjects editable field

## Side Effects

None expected.

## QA Notes

Should test editing subjects on the registries overview page, including:
- adding subject by browsing
- adding subject by searching
- remove subjects

 Cross-browser testing is recommended to make sure all styles are applied similarly across browsers.
